src/lando/ui/legacy/forms.py

```diff
@@ -12,8 +15,32 @@ class TransplantRequestForm(forms.Form):
 class UpliftRequestForm(forms.Form):
     """Form used to request uplift of a stack."""
 
-    revision_id = forms.RegexField(regex="D[0-9]+$")
-    repository = forms.CharField()
+    revision_id = forms.RegexField(
+        regex="D[0-9]+$",
```

Comment:
We can make the RE stricter.

```suggestion
        regex="^D[0-9]+$",
```